lib/src/oauth2_flows/metadata_server.dart

```diff
@@ -30,10 +32,13 @@ class MetadataServerAuthorizationFlow {
   factory MetadataServerAuthorizationFlow(http.Client client,
       {String email: 'default'}) {
     var encodedEmail = Uri.encodeComponent(email);
-    var scopesUrl =
-        Uri.parse('$_SERVICE_ACCOUNT_URL_PREFIX/$encodedEmail/scopes');
-    var tokenUrl =
-        Uri.parse('$_SERVICE_ACCOUNT_URL_PREFIX/$encodedEmail/token');
+
+    final metadataHost = Platform.environment[_GCE_METADATA_HOST_ENV_VAR] ??
```

Comment:
Missing the `dart:io` import. Is this class already limited to the VM?

Reply:
Thanks. Done - and yes, the class docstring is
```
/// Using this class assumes that the current program is running a
/// ComputeEngine VM. It will retrieve the current access token from the
/// metadata server.
```